tests/functional_tests/test_kask_w_cassandra.py

```diff
@@ -0,0 +1,58 @@
+import unittest, json, string, random
```

Comment:
```suggestion
import json
import random
import string
import unittest
```
Another nit: Imports get changed a bunch, and it can help make diffs easier to reason about if the lines that change map 1:1 with the modules being imported.